From: YunQiang Su <syq@debian.org>
Date: Mon, 14 May 2018 16:16:18 +0800
Subject: Enable R2 to R6 emulator by default
Forwarded: not-needed

In upstream code, 'mipsr2emu' kernel option is needed
to enable R2 to R6 emulator.  Since we need r6 kernel
for our r2 port, let's always enable it.

--- a/arch/mips/kernel/mips-r2-to-r6-emul.c
+++ b/arch/mips/kernel/mips-r2-to-r6-emul.c
@@ -56,7 +56,7 @@ extern const unsigned int fpucondbit[8];
 
 #define MIPS_R2_EMUL_TOTAL_PASS	10
 
-int mipsr2_emulation = 0;
+int mipsr2_emulation = 1;
 
 static int __init mipsr2emu_enable(char *s)
 {
